src/formatDate.js

```diff
@@ -50,7 +50,54 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arrDate = date.split(fromFormat[3].toString());
+  const resultArray = [];
+
+  let year = 0;
+  let month = 0;
+  let day = 0;
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    if (fromFormat[i].includes('Y')) {
+      year = arrDate[i];
+    } else if (fromFormat[i].includes('M')) {
+      month = arrDate[i];
+    } else if (fromFormat[i].includes('D')) {
+      day = arrDate[i];
+    }
```

Comment:
- It's better to use `switch` instead of checking just one letter.
- And Y/M/D, etc. looks like magic values